guiEditor/src/components/propertyTab/propertyTabComponent.tsx

```diff
@@ -173,7 +173,7 @@ export class PropertyTabComponent extends React.Component<IPropertyTabComponentP
                         }
                         resolve(adt.snippetId);
                     } else {
-                        reject();
+                        reject("string");

```

Comment:
A real error message even generic might be better here ?

Reply:
Whoops. Good find. ;)